src/App.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
 import './App.css';
+import BookList from './components/booklist';
+import BooksForm from './components/booksform';
 // import PropTypes from 'prop-types';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <p>
-          Hello world
-        </p>
-      </header>
+      <BookList />
```

Comment:
- This file should live in the `src/components` folder
- Remove comments from this file